application/src/main/java/net/corda/v5/application/crypto/DigestService.java

```diff
@@ -33,6 +33,15 @@ public interface DigestService {
     @NotNull
     SecureHash hash(@NotNull InputStream inputStream, @NotNull DigestAlgorithmName digestName);
 
+    /**
+     * Creates a {@link SecureHash}.
+     * <p>
+     * This function does not validate the length of the created digest. Prefer using
+     * <code>HashingService#parse</code> for a safer mechanism for creating {@link SecureHash}es.
```

Comment:
Is there a HashingService? I couldn't find any!

Reply:
Thank you, removed